packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -388,7 +389,7 @@ export class ActiveWorkflowRunner {
 				responsePromise?: IDeferredPromise<IExecuteResponsePromiseData>,
 				donePromise?: IDeferredPromise<IRun | undefined>,
 			): void => {
-				this.logger.debug(`Received trigger for workflow "${workflow.name}"`);
+				this.logger.warn(`Received trigger for workflow "${workflow.name}"`);
```

Comment:
This change will make the logs quite verbose IMO

Reply:
Did this on purpose - this highlighting makes finding relevant logs while testing much easier. They're only temporary, see above:

> Switched some logs to warn to make them easier to scan while debugging. Will revert these before merging.